Source/WebCore/rendering/TextBoxPainter.cpp

```diff
@@ -754,30 +755,30 @@ void TextBoxPainter<TextBoxPath>::paintPlatformDocumentMarker(const MarkedText&
 
     auto bounds = calculateDocumentMarkerBounds(makeIterator(), markedText);
 
-    auto lineStyleForMarkedTextType = [&]() -> DocumentMarkerLineStyle {
-        bool shouldUseDarkAppearance = m_renderer.useDarkAppearance();
+    auto lineStyleMode = [&]() -> DocumentMarkerLineStyleMode {
```

Comment:
is the `() -> DocumentMarkerLineStyleMode` needed?

Reply:
Removed.